source/base/exceptions.cc

```diff
@@ -484,6 +484,72 @@ namespace deal_II_exceptions
         }
     }
 
+
+
+#ifdef DEAL_II_WITH_CUDA
+    std::string get_cusparse_error_string(cusparseStatus_t error_code)
+    {
+      std::string message;
+      switch (error_code)
+        {
+        case CUSPARSE_STATUS_NOT_INITIALIZED:
+        {
+          message = "The cuSPARSE library was not initialized";
+
+          break;
```

Comment:
you could just do `return "...";` here and in all of the other places. saves you two lines each :-)